index.d.ts

```diff
@@ -0,0 +1,59 @@
+import { ViewStyle, LayoutAnimationConfig } from "react-native";
+import { ReactNode } from "react";
+
+interface KeyboardAccessoryViewProps {
+  style?: ViewStyle;
+  animateOn?: "ios" | "android" | "all" | "none";
+  animationConfig?: (() => LayoutAnimationConfig) | LayoutAnimationConfig;
+  alwaysVisible?: boolean;
+  bumperHeight?: number;
+  visibleOpacity?: number;
+  onKeyboardShowDelay?: boolean | number;
+  hiddenOpacity?: number;
+  hideBorder?: boolean;
+  inSafeAreaView?: boolean;
+  androidAdjustResize?: boolean;
+  avoidKeyboard?: boolean;
+  children:
+    | (({ isKeyboardVisible }: { isKeyboardVisible: boolean }) => ReactNode)
```

Comment:
Maybe it will be better to describe ```interface``` for such things, something like this

```ts
interface chidrenCallbackProps {
    isKeyboardVisible: boolean
}
```
What do you think?